packages/gdl-frontend/pages/games/browse.js

```diff
@@ -0,0 +1,245 @@
+      lang: string
+    }
+  },
+  intl: intlShape
+};
+
+class BrowsePage extends React.Component<Props> {
+  static async getInitialProps({ query, asPath, apolloClient, req }: Context) {
+    try {
+      await apolloClient.query({
+        query: BROWSE_GAMES_QUERY,
+        variables: {
+          page: INITIAL_PAGE_NUMBER,
+          language: query.lang,
+          pageSize: PAGE_SIZE
+        }
+      });
+      return {};
+    } catch (error) {
+      /*
+       * If user request invalid query param to graphql you trigger bad input validation
+       * and receive 400: Bad Request. The right feedback to the client is a 404 page
+       * and since graphql does not have a better error handling mechanism this is a dirty check.
+       */
+      if (
+        error.graphQLErrors &&
+        error.graphQLErrors.length > 0 &&
+        error.graphQLErrors[0].message === '400: Bad Request'
+      ) {
+        return {
+          statusCode: 404
+        };
+      }
+      return {
+        statusCode: 500
+      };
+    }
+  }
+
+  /**
+   * Load more games when demanded
+   */
+  handleFetchMore = (currentPage: number, fetchMore) => {
+    logEvent('Navigation', 'More - Browse', 'Games');
+
+    fetchMore({
+      variables: {
+        page: currentPage + 1
+      },
+      updateQuery: (
+        prev: BrowseGames,
+        { fetchMoreResult }: { fetchMoreResult: BrowseGames }
+      ) => {
+        if (!fetchMoreResult) return prev;
+        // Focus the first game of the extra games we're loading
+        const toFocus = fetchMoreResult.games_v2.results[0];
+        // Use a query selector to find the game we want to focus.
+        const gameAnchor = document.querySelectorAll(
+          `[href='/${toFocus.language}/games/details/${toFocus.id}']`
+        )[1];
+        gameAnchor && gameAnchor.focus();
+
+        return Object.assign({}, prev, {
+          games_v2: {
+            ...prev.games_v2,
+            pageInfo: fetchMoreResult.games_v2.pageInfo,
+            results: [
+              ...prev.games_v2.results,
+              ...fetchMoreResult.games_v2.results
+            ]
+          }
+        });
+      }
+    });
+  };
+
+  render() {
+    const {
+      router: { query },
+      intl
+    } = this.props;
+
+    return (
+      <Query
+        query={BROWSE_GAMES_QUERY}
+        variables={{
+          page: INITIAL_PAGE_NUMBER,
+          language: query.lang,
+          pageSize: PAGE_SIZE
+        }}
+      >
+        {({
+          loading,
+          error,
+          data,
+          fetchMore
+        }: {
+          data: BrowseGames,
+          loading: boolean,
+          error: any,
+          fetchMore: ({}) => void
+        }) => {
+          const {
+            games_v2: { pageInfo, results }
+          } = data;
+
+          return (
+            <Layout>
+              <Head
+                title={intl.formatMessage({
+                  id: 'Browse games',
+                  defaultMessage: 'Browse games'
```

Comment:
Må huske å legge disse inn i `site-translations-service` og i `crowdin`

Reply:
Hva er konsekvensen av å ikke legge inn i `site-translations-service` og `crowdin` først? Lurer på om vi ønsker å gjøre dette til en avhengighet eller at det kan operere som et eget issue. For jeg antar det er `defaultMessage` som blir vist?

Reply:
Er `defaultMessage` som vises da ja. Så sålenge vi ikke har noe annet språk enn engelsk er det jo ikke noe problem.

Reply:
https://github.com/GlobalDigitalLibraryio/site-translations-service/pull/5

Du får vise meg hvordan det gjøres med `crowdin` :)